homeassistant/components/qvr_pro/camera.py

```diff
@@ -0,0 +1,85 @@
+"""Support for QVR Pro streams."""
+
+import logging
+
+from pyqvrpro.client import QVRResponseError
+
+from homeassistant.components.camera import SUPPORT_STREAM, Camera
+
+from .const import DOMAIN, SHORT_NAME
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the QVR Pro camera platform."""
+    if discovery_info is None:
+        return
+
+    client = hass.data[DOMAIN]["client"]
+
+    entities = []
+
+    for channel in hass.data[DOMAIN]["channels"]:
+        entities.append(QVRProCamera(**channel, client=client))
+
+    add_entities(entities)
+
+
+class QVRProCamera(Camera):
+    """Representation of a QVR Pro camera."""
+
+    def __init__(self, name, model, brand, channel_index, guid, client):
+        """Init QVR Pro camera."""
+
+        self._name = f"{SHORT_NAME} {name}"
+        self._model = model
+        self._brand = brand
+        self.index = channel_index
+        self.guid = guid
+        self._client = client
+
+        try:
+            self._stream_source = self._client.get_channel_live_stream(guid)
+        except QVRResponseError as e:
```

Comment:
To solve the pylint error:
```suggestion
        except QVRResponseError as ex:
```